### PR TITLE
Don't fetch the model from aws s3

### PR DESCRIPTION
**Pull request by dbason**

### CODE REVIEW

**Review by tybalex — DISMISSED** [review]
lgtm

File: nulog-inference-service/start-nulog-inference.py [hunk 1]
```diff
@@ -376,17 +345,19 @@ async def schedule_update_pretrain_model(logs_queue):
     loop.run_until_complete(task)
 
     if IS_CONTROL_PLANE_SERVICE:
-        init_model_task = loop.create_task(get_pretrain_model())
-        loop.run_until_complete(init_model_task)
+        init_model_task = loop.create_task(get_pretrain_model(logs_queue))
+        model_loaded = loop.run_until_complete(init_model_task)
+        if not model_loaded:
+            os._exit(1)
```
Comment (sanjay920) [inline nulog-inference-service/start-nulog-inference.py]: I suggest `sys.exit()` over `os._exit()`. any reason why you went with the latter?

**Review by sanjay920 — COMMENTED** [review]
see comment

File: nulog-inference-service/start-nulog-inference.py (hunk 1 above)
Comment (dbason) [reply nulog-inference-service/start-nulog-inference.py]: Only because it lines up with what we do in go.  I have no strong preference.

**Comment by dbason** [comment]
Just pushed a couple of minor changes addressing your comment @sanjay920 , and removing the reload signal.  The signal isn't needed because when we first extract the model we haven't started inferencing

**Review by AmartC — APPROVED** [review]
Tested out the changes and everything is running smoothly. LGTM

**Pull request merged**